csrc/transformer/inference/csrc/apply_rotary_pos_emb.cu

```diff
@@ -113,6 +114,20 @@ __global__ void apply_rotary_pos_half(T* mixed_query,
     } else {                                    \
```

Comment:
@jagadish-amd Looks like there's more repetition here than necessary. If we really need to limit the 64 case to HIP/AMD, we can have just that condition ifdef'd. Right, @tjruwase?

Reply:
I see your point @jithunnair-amd. But I am fine either way.

Reply:
cannot have ifdef inside define.